src/main/java/uz/enterprise/mytex/entity/GroupPermission.java

```diff
@@ -0,0 +1,30 @@
+package uz.enterprise.mytex.entity;
+
+import lombok.*;
+import uz.enterprise.mytex.entity.audit.Auditable;
+
+import javax.persistence.*;
+
+/**
+ * @author - 'Zuhriddin Shamsiddionov' at 4:08 PM 10/24/22 on Monday in October
+ */
+@Entity
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "group_permissions")
+public class GroupPermission extends Auditable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @JoinColumn(name = "group_id", nullable = false)
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```
Follow this structure when writing relationship annotations
1. JoinColumn
2. ManyToOne
3. Always use FetchType.LAZY for many to one relationship, it will prevent N+1 performance issues